formatTest/unit_tests/expected_output/features403.re

```diff
@@ -13,7 +13,11 @@ type nonrec u 'a =
 type expr 'a =
   | Val {value: 'a} :expr 'a
   | Add
-      {left: expr int, right: expr int} :expr int
+      {
```

Comment:
Can the same `{` lifting to the previous line happen here?  It would be more consistent with the rest of the styling.